db/seeds.rb

```diff
@@ -5,5 +5,29 @@
     password: Faker::Internet.password(8),
     phone_number: Faker::Number.number(10),
     gender: "Male"
+  )
+
+  2.times do
+    Bot.create(
+    name: Faker::Name.first_name,
+    gender: 'Female',
+    eye_color: 'Green',
+    hair_color: 'Black',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.